engine/apps/integrations/mixins/alert_channel_defining_mixin.py

```diff
@@ -67,18 +78,26 @@ def dispatch(self, *args, **kwargs):
                         alert_receive_channel = obj.object
```

Comment:
hm.. Not to change anything now (and unrelated to your changes), but I think this json deserialization + loop above this line could be improved.